catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/QueryResponseImpl.java

```diff
@@ -50,6 +50,8 @@
 
     List<Result> resultList = null;
 
+    private long timeoutMillis = 30000;
```

Comment:
You've got inconsistent defaults of 30s here and in `QueryOperations` but 300s in your blueprint/metatype.